multimedia/video-compare/Portfile

```diff
@@ -0,0 +1,36 @@
+# Portfile for video-compare
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        pixop video-compare 20250928
+github.tarball_from archive
+
+# Port metadata
+revision            0
+categories          multimedia
+maintainers         nomaintainer
+license             GPL-2.0
+description         A tool for comparing video files visually.
+long_description    Split screen video comparison tool using FFmpeg and SDL2
+
+
+checksums           rmd160  2c55262f1692237a0b5528e6aea6748064e9d4b3 \
+                    sha256  cdcdb764868cf358ef5337f68e7bfd4526ec3efac27cc35db70c4158e3dea99f \
+                    size    1184614
+
+
+depends_lib         port:ffmpeg \
+                    port:libsdl2 \
+                    port:libsdl2_ttf
+
+# Build environment
+build.env           PATH=${prefix}/bin:${env(PATH)}
```

Comment:
this shouldn't bee needed - what's the problem you're trying to solve and/or error message?